activerecord/lib/arel/nodes/optimizer_hints.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module Arel # :nodoc: all
+  module Nodes
+    class OptimizerHints < Arel::Nodes::Node
```

Comment:
There is no reason to touch the `OptimizerHints` node in this PR anymore.

Reply:
`OptimizerHints` is no longer impacted by this PR. :+1: